lib/networkPacks/PacksForServer.h

```diff
@@ -429,6 +429,31 @@ struct DLL_LINKAGE ManageBackpackArtifacts : public CPackForServer
 	}
 };
 
+struct DLL_LINKAGE ManageEquippedArtifacts : public CPackForServer
+{
+	ManageEquippedArtifacts() = default;
+	ManageEquippedArtifacts(const ObjectInstanceID & artHolder, const size_t costumeIdx, bool saveCostume = false)
+		: artHolder(artHolder)
+		, costumeIdx(costumeIdx)
+		, saveCostume(saveCostume)
+	{
+	}
+
+	ObjectInstanceID artHolder;
+	size_t costumeIdx;
```

Comment:
Please use only fixed-width types in serialization (e.g. uint8_t / uint32_t / etc, whatever is suitable).
This is required for cross-platform gameplay